### PR TITLE
Ng 12 show memory

### PR DESCRIPTION
**Pull request by alejoreyes96**
Added memory tables to display.

### CODE REVIEW

**Review by noelvalentin — APPROVED** [review]

File: GUI/window.py [hunk 1]
```diff
@@ -80,46 +80,61 @@ def __init__(self, **kwargs):
...
+        self.reg_table.get_data()
+        self.mem_table.data_list.clear()
+        self.mem_table.get_data()
+        
+        self.add_widget(self.reg_table)
+        self.add_widget(self.mem_table)
 
     def run_micro_instructions(self, instance):
-        if not self.micro_sim.is_running:
+        if not(self.micro_sim.is_running):
```
Comment (cfboy) [inline GUI/window.py]: Remove parenthesis.

File: GUI/window.py [hunk 2]
```diff
@@ -80,46 +80,61 @@ def __init__(self, **kwargs):
...
+        self.reg_table.data_list.clear()
+        self.reg_table.get_data()
+        self.mem_table.data_list.clear()
+        self.mem_table.get_data()
+
         toast('Micro memory cleared! Load new data')
 
     def run_micro_instructions_step(self, instance):
-        if not self.micro_sim.is_running:
+        if not(self.micro_sim.is_running):
```
Comment (cfboy) [inline GUI/window.py]: Remove parenthesis.

File: GUI/window.py [hunk 3]
```diff
@@ -215,24 +229,24 @@ def get_data(self):
         self.data = [{"text": str(x.upper()), "color": (.1, .1, .1, 1)} for x in self.data_list]
 
 
-class Table2(RecycleView):
+class MemoryTable(RecycleView):
     data_list = ListProperty([])
 
     def __init__(self, **kwargs):
-        super(Table2, self).__init__(**kwargs)
+        # self.register = REGISTER
```
Comment (ChristianPerez34) [inline GUI/window.py]: Remove comment

**Review by ChristianPerez34 — CHANGES_REQUESTED** [review]
Minor details

File: GUI/window.py (hunk 1 above)
Comment (alejoreyes96) [reply GUI/window.py]: done

File: GUI/window.py (hunk 2 above)
Comment (alejoreyes96) [reply GUI/window.py]: done

File: GUI/window.py (hunk 3 above)
Comment (alejoreyes96) [reply GUI/window.py]: done

**Review by ChristianPerez34 — APPROVED** [review]

**Review by cfboy — APPROVED** [review]

**Pull request merged**